### ISSUE TITLE
Fix label_molecule.py on topology branch

### ISSUE THREAD

**Issue by proteneer** [openforcefield/openforcefield#187]
Some functions have been moved around
``` python
from openforcefield.utils import *
from openforcefield.utils.structure import get_molecule_parameterIDs
from openforcefield.typing.engines.smirnoff import ForceField

from oechem import OEMol, OEParseSmiles, OEAddExplicitHydrogens
```

**Comment by jchodera** [comment]
@proteneer : Can you clarify this issue? What exactly is the problem you're reporting here?

**Comment by proteneer** [comment]
The current topology branch shows this:

https://github.com/openforcefield/openforcefield/blob/topology/examples/label_molecule/label_molecule.py

Which is *completely* broken.

**Comment by davidlmobley** [comment]
I think the issue is just that the examples haven't been updated yet. Or am I missing something?

**Comment by proteneer** [comment]
That's pretty much it.

**Comment by j-wags** [comment]
We've started taking steps towards a fix in 7de10a3. We'll start on a complete fix after #173 is merged because it makes such significant changes to how parameters are handled.

**Comment by j-wags** [comment]
#225 closes this
